test/e2e/provider_expectations.go

```diff
@@ -217,7 +217,7 @@ func (tc *testConfig) ExpectNodeToBeDrainedBeforeDeletingMachine() error {
 		}
 		for _, machineItem := range machineList.Items {
 			// empty or non-worker role skipped
-			if machineItem.Labels["sigs.k8s.io/cluster-api-machine-role"] == "worker" {
+			if machineItem.Labels["machine.openshift.io/cluster-api-machine-role"] == "worker" {
```

Comment:
this will never pass the tests unless you make it backward compatible with the installer labels